collector_module/opmon_collector/collector_worker.py

```diff
@@ -138,6 +144,32 @@ def _parse_attachment(self, opmon_response):
             self.log_warn("Cannot parse response attachment.", '')
             raise e
 
+    def _store_records_to_file(self, repeat):
+        if len(self.records):
+            host_name = re.sub("[^0-9a-zA-Z.-]+", '.', self.server_data['server'])
+
+            now = datetime.datetime.now()
+            log_path = f"{self.settings['collector']['documents-log-directory']}" \
+                       f"/{self.settings['xroad']['instance']}/{now.year:04d}/{now.month:02d}/{now.day:02d}/"
+
+            if not os.path.exists(log_path):
+                os.makedirs(log_path)
+
+            self.log_info(f"Appending {len(self.records)} documents to log file.")
+            records_logger = logging.getLogger(host_name)
+            records_logger.setLevel(logging.INFO)
+            # Adding handler only if this is not a repeated query to the host
+            if repeat == 0:
+                handler = RotatingFileHandler(
+                    log_path + host_name + ".log",
+                    maxBytes=self.settings['collector'].get('documents-log-file-size', 0),
```

Comment:
Small suggestion - I'd maybe have this as:
```suggestion
                    maxBytes=self.settings['collector'].get('documents-log-file-size') or 0,
```
This would also assure us that in case in settings.yaml the `documents-log-file-size` was set to None - we'd still have 0 as a value. Does that make sense?